src/main/controlMod.F90

```diff
@@ -359,8 +359,8 @@ subroutine control_init( )
                errMsg(sourcefile, __LINE__))
        end if
 
-       if (n_dom_soil_patches < 0) then
-          call endrun(msg=' ERROR: n_dom_soil_patches must be at least 0 ' // &
+       if (n_dom_soil_patches <= 0) then
+          call endrun(msg=' ERROR: expecting n_dom_soil_patches between 1 and 16' // &
                errMsg(sourcefile, __LINE__))
```

Comment:
Minor correction.

Reply:
Now my standard testing fails. I think that n_dom_soil_patches is set to 0 when I do not specify a default value in the namelist. So I'm testing now with a different default value in the namelist.